CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ If you have not done so on this machine, you need to:
  
 * Install Git and configure your GitHub access
 * Install Java SDK (OpenJDK recommended)
-* Install [GraalVM](http://www.graalvm.org/downloads/) (community edition is enough)
+* Install [GraalVM](https://quarkus.io/guides/building-native-image )
```

Comment:
```suggestion
* Install [GraalVM](https://quarkus.io/guides/building-native-image)
```
(there is a trailing space at the end of the link, let's get rid of it)

Reply:
updated